content/posts/2025-10-02-WildFly38Beta-released.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title:  "New WildFly 38 Beta release"
+aliases: [/news/2025/10/02/WildFly38Beta-released/]
+date:   2025-10-02
+tags:   announcement release
+author: darranl
+description: WildFly 38 Beta is now available for download.
+---
+
+We're excited to announce the release of *WildFly 38 Beta 1*! This version is contains updates, new features, and bug fixes, paving the way for the upcoming final release. This beta version is a crucial step in our development cycle, allowing the community to test and provide feedback on the latest changes.
+
+
+== What's New and Noteworthy?
+
+This release focuses heavily on *component upgrades*, *MicroProfile 7.1 support*, and enhancements to the *WildFly Preview* distribution. Here's a quick rundown of the highlights:
```

Comment:
I'm not a fan of all this bold text. WDYT?

If I see bold text I either expect it to be emphasizing something that's not obviously a focal point of the text, or I expect it to be a hyperlink (i.e. maybe it's not really 'bold' it's just that link text is presented in a 'bolder' style.)